tests/integration/security/fuzz/fuzzers/dotdotpwn/Makefile

```diff
@@ -0,0 +1,20 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+HUB="gcr.io/ymzhu-istio/dotdotpwn"
```

Comment:
This should probably be pulling from an Istio repo.

Reply:
I have a TODO for this, please see my reply in your other comment.

Reply:
Thanks for addressing this in a later commit.